spec/models/ext_management_system_spec.rb

```diff
@@ -376,7 +417,7 @@
     end
 
     it "#total_vcpus" do
-      expect(@ems.total_vcpus).to eq(8)
+      expect(@ems.total_vcpus).to eq(16)
```

Comment:
would be nice if this value didn't change.

maybe add disk capacity and total cores to the same model?